qqc/pipeline.py

```diff
@@ -121,15 +126,16 @@ def dicom_to_bids_QQC(args, **kwargs) -> None:
         config.read(args.config)
         for root, dirs, files in os.walk(qqc_input):
             for subdir in dirs:
-                if 't1w_mpr_nd' in subdir.lower():
+                if is_xa30(qqc_input):
                     # TODO add enhanced logic
```

Comment:
Obsolete comment in line 130

Reply:
[2a5844e](https://github.com/AMP-SCZ/qqc/pull/52/commits/2a5844ea5a60809656ba5ef88086e1ea517596b5)